### ISSUE TITLE
Why can't the current RAP detect UAF cases in tests?

### ISSUE THREAD

**Issue by kynehc** [Artisan-Lab/RAP#83]
Hi, it seems the latest version (b98fef672817635d1d3706a04e415b07f77e6e9f) can't detect any UAF cases in the tests folder. Am I using it incorrectly?

I tried different log levels. Below is the default output log for uaf_small (refer to the case in https://artisan-lab.github.io/RAP-Book/6.1-dangling.html).
```
11:19:43|RAP|INFO|: Start cargo-rap
11:19:43|RAP|INFO|: cargo clean in package folder .
11:19:43|RAP|INFO|: cargo check in package folder .
    Checking uaf_small v0.1.0 (/rust/rap/RAP/tests/uaf_small)
11:19:43|RAP|INFO|: Execute after_analysis() of compiler callbacks
11:19:43|RAP|INFO|: analysis done
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.35s
```

**Comment by hxuhack** [comment]
Thanks. We just noticed the issue caused by a version update and will fix it soon.

**Comment by VaynNecol** [comment]
@kynehc Thanks for your report. The hot fix can be seen in  [commit fa8854f](https://github.com/Artisan-Lab/RAP/commit/fa8854f51d6a22dc7a5fb06d43543125f782cfc2).
